src/components/home/partners/secondaryPartnerList.js

```diff
@@ -15,14 +15,21 @@ export const secondaryPartnerList = [
       "Digital vaults are essential components of the future 'stack' that every individual needs to control their lives in our modern economy. CODE PDX's focus on this exemplifies their understanding of what is most important for the vulnerable in our society. The remarkable team at Code for PDX should be applauded for their energy and vision.",
     testimonialAuthor: '- Siva Narendra, Co-Founder ORDSN',
     testimonialAuthorTwo: '- Carol Coye Benson, Co-Founder ORDSN',
-    partnerLogo: '/assets/ordsnLogo.png',
+    partnerLogo: '/assets/partnerImages/ordsnLogo.png',
     website: 'https://www.ordsn.org/'
   },
   {
     company: 'Epicodus',
     testimonial:
-      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam, quaerat saepe possimus ex dolor deleniti excepturi alias, est in tempore similique, inventore doloribus voluptatibus voluptate. Reiciendis minima nihil et dicta.',
-    partnerLogo: '/assets/epicodusLogo.png',
+      "Epicodus's mission is to help people learn the skills they need to get great jobs. For us, “great jobs” means jobs in growing industries that pay well and provide rewarding work. Beyond the particular skills needed to get these jobs, we aim to help our students become confident self-teachers who can adapt to changing job markets, and great communicators who will work well in teams. We focus on serving people who, by birth or circumstance, don't have easy access to learning the skills they need to get great jobs..",
+    partnerLogo: '/assets/partnerImages/epicodusLogo.png',
     website: 'https://www.epicodus.com/'
+  },
+  {
+    company: 'CETI',
+    testimonial:
+      'A Creative and Emergent Technology Institute, CETI champions lifelong learning and collaborative, interdisciplinary innovation in creative and emergent technologies',
```

Comment:
I'd prefer either the **Mission** section of their `About` page:
```
CETI champions lifelong learning and collaborative, interdisciplinary innovation in creative and emergent technologies. Through radically inclusive community participation, we rapidly build expertise for an astonishing diverse group of people. CETI contends that our most urgent challenges can be imaginatively addressed through art, storytelling and invention.
```

OR

the **Who We Are** section of their `About` page:
```
CETI prioritizes including and collaborating with communities previously marginalized, under-represented, and under-served  in tech. Through our lived experiences and diversity in demographics, we bring a diversity of critical and cultural lenses to the intentional design of our programs, as well as a laser-focus on inclusion and support of communities significantly under-represented in technology conversations.  

CETI is a collective—of artists, technologists, storytellers, educators, researchers, innovators, and leaders—composed of experts and lifelong learners, excited to share our ideas and skills. A subset of us engage with each project, alternatively taking on roles as learners, facilitators, creators, producers, and advisors, wielding the powerful tools of art, emerging technology, and storytelling  to imagine and create inclusive, just futures.
```